Lombiq.Tests.UI/Extensions/ScrollingUITestContextExtensions.cs

```diff
@@ -47,4 +47,59 @@ public static Point GetScrollPosition(this UITestContext context)
             Convert.ToInt32(position[0], CultureInfo.InvariantCulture),
             Convert.ToInt32(position[1], CultureInfo.InvariantCulture));
     }
+
+    /// <summary>
+    /// Hides browser scroll-bars if they were not hidden with <see cref="HideScrollBar(UITestContext)"/> previously.
```

Comment:
As I see, both "scroll bar" and "scrollbar" go, but not "scroll-bar".

Also, since "scrollbar" seems to be the more prevalent, please rename "crollBar" -> "crollbar".